### DISCUSSION TITLE
How would I set the HmIP-BWTH Mode to Cooling?

### DISCUSSION THREAD

**Discussion by macrojames** [SukramJ/aiohomematic#1124] [Q&A] [answered]
Currently I am using a CCU program to switch specific devices between cooling and heating, depending on the outside temperature and the state of my heatpump.
How could i achieve that in HA Automations? 
I read about activating disabled entities but couldn't find anything matching the hvac mode in the HA GUI
It looks like those are parameters to set on channel :1 but I don't unterstand what param or value I need to set here.

In the WebUI of CCU some values are visible in the Selectbox.
![image](https://github.com/danielperna84/hahomematic/assets/482151/59e77c77-6d52-4ab6-b262-099a0727beeb)

Any hint is appreciated.

**Comment by SukramJ** [comment]
[See](https://github.com/danielperna84/hahomematic/discussions/1022#discussioncomment-5621988)

**Comment by macrojames** [comment, marked as answer]
Perfect.
Although I already read that post I didn't understand it correctly the first time.

So a more verbose version for future questions:
- Create a file in your `/config/homematicip_local/` directory, named `unignore`
- Edit the file `unignore` and literally add `HEATING_COOLING` to it, save it.
- Restart your Home Assistant
- An new select entity is created and added to every device supporting it.
- You can not switch the mode in the climate card directly, you must use this select entity

Thankyou @SukramJ for your awesome work in creating, maintaining and updating this component. Your effort is highly appreciated! ❤
